changelog.d/463.bugfix

```diff
@@ -0,0 +1 @@
+Don't automatically join new users to all public channels
```

Comment:
#463 is not this PR but #463 added `changelog.d/431.bugfix`... okaaaaaayyyy.

Reply:
i think someone (me) might have left out a file from another PR :s